CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ All notable, unreleased changes to this project will be documented in this file.
 
 ### Highlights
 
+- Drop demo - #14835 by @fowczarek
```

Comment:
IMO, it's not a highlight, for current users of Saleor it shouldn't actually matter as they shouldn't be using demo mode; it's more an internal change.